dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/actionlet/FourEyeApproverActionletTest.java

```diff
@@ -83,7 +83,7 @@ public static void prepare() throws Exception {
         contentletAPI = APILocator.getContentletAPI();
         languageAPI = APILocator.getLanguageAPI();
 
-        site = new SiteDataGen().nextPersisted();
+        site = new SiteDataGen().name("site"+System.currentTimeMillis()).nextPersisted();
```

Comment:
A unique name should happen automatically when you call `new SiteDataGen().nextPersisted();` If it doesn't, this code should be moved there.

Reply:
You right fixed